### PR TITLE
fix issue #126

### PR DESCRIPTION
**Pull request by richban**
- Created View to list all meet-ups
- Created template

### CODE REVIEW

File: konfera/templates/konfera/meetups.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{% extends 'konfera/base.html' %}
+{% load i18n %}
+
+{% block title %}
+  {% trans "List of upcoming meetups" %}
```
Comment (ricco386) [inline konfera/templates/konfera/meetups.html]: We do show also past meetups here, how about we change it to: *Meetups list*

File: konfera/urls.py [hunk 2]
```diff
@@ -17,6 +17,9 @@
 
 
 urlpatterns = [
+    url(r'^event/', include('konfera.event.urls')),
```
Comment (ZoltanOnody) [inline konfera/urls.py]: Remove this line. It's included on line 23

File: konfera/urls.py [hunk 3]
```diff
@@ -17,6 +17,9 @@
...
+    url(r'^meetups/', views.meetup_list, name='list_all_meetups'),
+    url(r'^(?P<event_slug>[\w, -]+)/register/volunteer/$', views.register_volunteer, name='event_register_volunteer'),
```
Comment (ZoltanOnody) [inline konfera/urls.py]: Remove this line. It was moved to `konfera.register.urls`.

File: konfera/register/views.py [hunk 4]
```diff
@@ -1,4 +1,10 @@
 from django.shortcuts import render, get_object_or_404, redirect
+from django.utils.translation import ugettext_lazy as _
+
+from konfera.event.forms import SpeakerForm, TalkForm
+from konfera.forms import VolunteerRegistrationForm
+from konfera.models.event import Event, MEETUP
+from konfera.models.talk import CFP
```
Comment (ZoltanOnody) [inline konfera/register/views.py]: You add some unused imports. Please remove them.

File: konfera/urls.py (hunk 3 above)
Comment (payowallet) [inline konfera/urls.py]: Do you mean all 3 lines?

Comment (richban) [inline konfera/urls.py]: do you mean all 3 lines?

Comment (ZoltanOnody) [inline konfera/urls.py]: Just the last one.

**Comment by ZoltanOnody** [comment]
Resolve the conflicts with master and it's good to go. :)

**Pull request merged**